cmd/state-installer/test/integration/installer_int_test.go

```diff
@@ -41,8 +42,8 @@ func (suite *InstallerIntegrationTestSuite) TestInstallFromLocalSource() {
 	suite.NotContains(cp.TrimmedSnapshot(), "Downloading State Tool")
 
 	// Assert expected files were installed (note this didn't use an update payload, so there's no bin directory)
-	suite.FileExists(appinfo.StateApp(target).Exec())
-	suite.FileExists(appinfo.SvcApp(target).Exec())
+	suite.FileExists(filepath.Join(target, installation.BinDirName, constants.StateCmd+osutils.ExeExt))
+	suite.FileExists(filepath.Join(target, installation.BinDirName, constants.ServiceCommandName+osutils.ExeExt))
```

Comment:
Why not use appinfo here?

Reply:
Because of the test conditional, we end up with an `appinfo` from the build dir and not the one that was just installed

Reply:
Yeah, I'm working on a fix for it.